labrad/types.py

```diff
@@ -295,8 +295,20 @@ def unflatten(s, t, endianness='>'):
     return t.__unflatten__(s, endianness)
 
 
-FlatData = collections.namedtuple('FlatData', ['bytes', 'tag'])
+class FlatData(collections.namedtuple('FlatDataBase', ['bytes', 'tag', 'endianness'])):
+    """FlatData represents unflattened data.
 
+    Attributes:
+
+    bytes:      The data as intended to be transmitted over the wire
+    tag:        The LabRAD data type in the form of an LRType object.
+    endianness: The endiannes of the connection. '>' for big endian
+                and '<' for little endian.  Little endian is deprecated.
```

Comment:
nit: docs of individual attributes should be indented under the `Attributes:` section.